pkg/metadata/metadata.ipldsch

```diff
@@ -0,0 +1,5 @@
+type ContentClaimMetadata struct {
+  ClaimType Int
+  ShortCut Bytes
```

Comment:
🙏 can this be named better? Something like "embedded data"/"cached data"/"claimed data"?

Reply:
fair